scripts/export.py

```diff
@@ -157,6 +167,9 @@ def nemo_export(argv):
             )
         )
         raise e
+    finally:
+        if export_cache_support_prev is not None:
```

Comment:
After export, model is considered corrupted, cannot be used for any purpose. So this is not needed.

Reply:
What do you mean it is corrupted?

Reply:
NeMo export prep does things to model graph that cannot be reasonably undone. So yes, model is generally not usable after export.

Reply:
dropped it.